src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -14,27 +16,102 @@ namespace System.Net.Security.Tests
 {
     public class SslStreamStreamToStreamTest
     {
+        private readonly byte[] sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+        private const int TaskWaitTime = 15; // seconds
+
+        #region Tests
+
         [Fact]
         public void SslStream_StreamToStream_Authentication_Success()
         {
+            SslStream clientSslStream = null;
+            SslStream serverSslStream = null;
+            FakeNetworkStream clientStream = null;
+            FakeNetworkStream serverStream = null;
+
+            try
+            {
+                bool result = GetServerClientSslStream(out clientSslStream, out serverSslStream, out clientStream, out serverStream, 3);
```

Comment:
Having so many parameters and so many of them marked as `out` is a [code smell](https://en.wikipedia.org/wiki/Code_smell). This particular case shows that the code doesn't encapsulate well and doesn't respect the single responsibility principle.